build.py

```diff
@@ -268,6 +268,8 @@ def clone_or_update_repo(name, branch, dir_name, owner='openwisp'):
         )
     # If the module contains a doc directory, copy it to the dir_name in the root.
     # Otherwise, copy the entire directory.
+    if os.path.exists(dir_name):
```

Comment:
Can you try with symbolic links instead of copying the entire dir to see if it works? If it does it should make the process faster.